octoprint/tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py{27,38}
```

Comment:
Were you able to test this with python 2.7? It may be less hassle just to have this for python 3.8.

Reply:
CI runs this with both